internals/0-fs.sh

```diff
@@ -0,0 +1,52 @@
+TF_HOME="${GOPATH}/src/github.com/threefoldtech"
+
+G8UFS_REPOSITORY="https://github.com/threefoldtech/0-fs"
```

Comment:
Maybe change G8UFS name here, since you moved to zfs (which is much better, thanks :p)
And everywhere else btw on the script